contrib/devtools/proto-tools-installer.sh

```diff
@@ -116,28 +122,33 @@ f_install_protoc_gen_swagger() {
 }
 
 f_install_clang_format() {
-  f_print_installing_with_padding clang-format
+  f_print_installing_with_padding ${CLANG_FORMAT_BIN}
 
-  if which clang-format &>/dev/null ; then
+  if which ${CLANG_FORMAT_BIN} &>/dev/null ; then
     echo -e "\talready installed. Skipping."
     return 0
   fi
 
   case "${UNAME_S}" in
   Linux)
     if [ -e /etc/debian_version ]; then
-      echo -e "\tRun: sudo apt-get install clang-format" >&2
+      echo -e "\tRun :\n wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add - ; \
+sudo apt-add-repository \"deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-6.0 main\"; \
```

Comment:
This is bad. On Debian/Ubuntu clang can be installed via the default repositories. You are recomending here an external repository regardless what distro the user is running. Please revert - feel free to enrich the command's output with references to external manuals that the users can consult.

Reply:
Yes, @akhilkumarpilli you don't need to add external repository as this is available via default repos.
```suggestion
      echo -e "\tRun: sudo apt-get install clang-format-6.0" >&2
```

Reply:
We need to add apt repository here to install `clang-format-6.0`.  And repository which we are adding here is actually the original repository where `clang-format` is developed from.

Reply:
What version of Ubuntu are you running? 'Cause since Xenial `clang-format-6` is supported. Please see: https://packages.ubuntu.com/search?keywords=clang-format-6

Reply:
Yes, `xenial` has support for `clang-format-6.0`. @akhilkumarpilli did you test it on your system? Are there any issues?

Reply:
Ohh, I am running `16.04.6 LTS xenial` and `clang-format-6.0` is not available directly.

Reply:
You need to enable the `universe` component of the repository.

Reply:
I think there are some issues in my system. I tried in other xenial version running system and it worked. Sorry for that. Thanks @alessio